openid-caep-interoperability-profile-1_0.md

```diff
@@ -125,7 +125,7 @@ The following requirements are common across all use-cases defined in this docum
 
 ## Network layer protection
 * The SSF transmitter MUST offer TLS protected endpoints and MUST establish connections to other servers using TLS. TLS connections MUST be set up to use TLS version 1.2 or later.
-* When using TLS 1.2, follow the recommendations for Secure Use of Transport Layer Security in [RFC7525]{{RFC7525}}.
+* When using TLS 1.2 or later, follow the recommendations for Secure Use of Transport Layer Security in [RFC9325]{{RFC9325}}.
```

Comment:
As only 1.2 or later can be used, probably don't need that qualifier, I'd suggest:

```suggestion
* The SSF transmitter MUST follow the recommendations for Secure Use of Transport Layer Security in [RFC9325]{{RFC9325}}.
```

Reply:
I adapted the change. Doesn't the same apply for the receiver?

Reply:
oh good point! I guess it should have an 'and receiver'?